src/main/java/core/basesyntax/vehicles/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
```

Comment:
The message 'Bulldozer started working' could be more informative by specifying that it is a Bulldozer machine. Consider rephrasing it to include the machine type explicitly.